falcon_multiqc/commands/connect.py

```diff
@@ -0,0 +1,69 @@
+import click
+from getpass import getpass
+from sqlalchemy import create_engine, inspect
+from sqlalchemy.exc import OperationalError
+
+# connect command allows user to modify the config.py DATABASE_URI with their username, password and database name
+# if the user has not yet made a falcon_multiqc database, this command allows them to make a new one
+
+#global list 
+falcon_multiqc_schema = ['cohort', 'patient', 'batch', 'sample', 'PatientBatch', 'raw_data']
```

Comment:
Instead of this, can we use Base to somehow do the same thing here? So if we change it, we don't have to update it here. And also I'm getting issues with it saying my db is not a falcon_multiqc db.

Reply:
That message should only be raised if the database name you entered doesn't have the same matching tables as the falcon_multiqc databases, is that not the case?  Then that's weird ...

Reply:
Btw, in terms of the 'cannot import name 'DATABASE_URI' from 'database.config'' that's happening because from now on the first thing you have to do is use the connect command to log into the database and remake the empty config.py file

Reply:
But yeah good idea I'll do the Base thing

Reply:
I think you should add a function inside Models.py like getTables(). I just had a go at doing it, here's the code. If you have a better way for sure do it.

`               tables = []
                for name, model_class in Base._decl_class_registry.items():
                    try:
                        tables.append(model_class.__tablename__)
                    except:
                        # Sometimes there is an object that's not a model class in this iteration.
                        continue
               return tables`

Reply:
Sorry I can't format that properly in comments haha

Reply:
Nice I was trying to work out how to do it haha, okay that sounds good for now!